.gitignore

```diff
@@ -3,6 +3,7 @@
 .bundle
 .config
 .yardoc
+.idea
```

Comment:
This was for RubyMine's cache and metafiles, which are created automatically. You can safely ignore or pick around it.